clinica/iotools/utils/data_handling.py

```diff
@@ -235,6 +241,24 @@ def create_merge_file(
         cprint("End of CAPS information merge.", lvl="debug")
 
 
+def create_participants_df(bids_dir: str) -> pandas.DataFrame:
+    """
+    Function to create participant.tsv "on the fly" in case it is not found in the BIDS directory
+    """
+    import pandas as pd
```

Comment:
You should probably just `from pandas import DataFrame` for the type signature and use it inside the function without requiring another qualified import here.

Reply:
Removed. Wasn't used in data_handling